src/components/EditorsColumn.jsx

```diff
@@ -61,10 +62,7 @@ export default class EditorsColumn extends React.Component {
     const children = [];
     const [hiddenLanguages, visibleLanguages] = partition(
       ['html', 'css', 'javascript'],
-      language => includes(
-        currentProject.hiddenUIComponents,
-        `editor.${language}`,
-      ),
+      language => find(currentProject.hiddenUIComponents, {language}),
```

Comment:
@outoftime My only hesitation with renaming the `language` property in `HiddenUIComponent` to `instance` is this line, since I really like the how the `language` property connects to how language is used in this file. `instance` also wouldn't be used by anything that isn't an editor, and I can't imagine we'd want multiple consoles or instructions.